appveyor.yml

```diff
@@ -2,8 +2,18 @@ environment:
   matrix:
   - JULIAVERSION: "julialang/bin/winnt/x86/0.4/julia-0.4-latest-win32.exe"
   - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.4-latest-win64.exe"
```

Comment:
Let's drop 0.4 testing on App app veyor like we did on travis and a few lines down don't allow 0.6 tests to fail.